onyxia-api/src/main/java/fr/insee/onyxia/api/security/DisableCORS.java

```diff
@@ -1,15 +1,18 @@
 package fr.insee.onyxia.api.security;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @Configuration
 public class DisableCORS implements WebMvcConfigurer {
 
+    @Value("${cors.origins}")
+    private String corsOrigins;
+
     @Override
     public void addCorsMappings(CorsRegistry registry) {
-        registry.addMapping("/**").allowedMethods("*");
+        registry.addMapping("/**").allowedOrigins(corsOrigins).allowedMethods("*");

```

Comment:
This configuration does not allow users to disable CORS entirly. Not sure how this works if user set's corsOrigins to an empty string.